ktor-swagger-ui/src/main/kotlin/io/github/smiley4/ktorswaggerui/dsl/routing/resources/DocumentedResourceRoutes.kt

```diff
@@ -4,19 +4,16 @@ import io.github.smiley4.ktorswaggerui.dsl.routes.OpenApiRoute
 import io.github.smiley4.ktorswaggerui.dsl.routing.documentation
 import io.github.smiley4.ktorswaggerui.dsl.routing.method
 import io.ktor.http.HttpMethod
-import io.ktor.server.application.ApplicationCall
-import io.ktor.server.resources.handle
 import io.ktor.server.resources.resource
 import io.ktor.server.routing.Route
-import io.ktor.util.pipeline.PipelineContext
 
 //============================//
 //             GET            //
 //============================//
 
 inline fun <reified T : Any> Route.get(
     noinline builder: OpenApiRoute.() -> Unit = { },
-    noinline body: suspend PipelineContext<Unit, ApplicationCall>.(T) -> Unit
+    noinline body: suspend io.ktor.server.routing.RoutingContext.() -> Unit
```

Comment:
Out of curiosity, why the FQN here, is there a conflict with another class of the same name?

Reply:
do you mean the RoutingContext ?

Reply:
Yes.